stdlib/3/urllib/parse.pyi

```diff
@@ -123,11 +124,18 @@ def urldefrag(url: str) -> DefragResult: ...
 @overload
 def urldefrag(url: bytes) -> DefragResultBytes: ...
 
-def urlencode(query: Union[Mapping[Any, Any],
+if sys.version_info >= (3, 5):
+    def urlencode(query: Union[Mapping[Any, Any],
                            Mapping[Any, Sequence[Any]],
                            Sequence[Tuple[Any, Any]],
                            Sequence[Tuple[Any, Sequence[Any]]]],
-              doseq: bool = ..., safe: AnyStr = ..., encoding: str = ..., errors: str = ...) -> str: ...
+                  doseq: bool = ..., safe: AnyStr = ..., encoding: str = ..., errors: str = ..., quote_via: Callable[[str, AnyStr, str, str], str] = ...) -> str: ...
```

Comment:
Can you reflow this line to be a bit shorter (< 100)? Also the previous three lines should be indented 4 spaces more.